backend/database/seeding/hp_seed_data.go

```diff
@@ -361,3 +364,11 @@ var FindingBook2Robes = newHPFinding(OpChamberOfSecrets.ID, "find-uuid-robes", n
 
 var newHPServiceWorker = newServiceWorkerGen(1)
 var DemoServiceWorker = newHPServiceWorker("Demo", `{ "type": "web",  "version": 1, "url": "http://demo:3001/process" }`)
+
+var newGlobalVar = newGlobalVarGen(1)
+var VarExpelliarmus = newGlobalVar("Expelliarmus", "disarm an opponent")
+var VarAlohomora = newGlobalVar("Alohomora", "unlock doors")
+var VarAscendio = newGlobalVar("Ascendio", "lifts the caster high into the air")
+var VarImperio = newGlobalVar("Imperio", "control another person")
+var VarLumos = newGlobalVar("Lumos", "creates a narrow beam of light")
+var VarObliviate = newGlobalVar("Obliviate", "erases memories")
```

Comment:
:+1:

Reply:
haha I'm all about 1) honoring convention and 2) referencing media franchises from my childhood in my work